consumers/consumer.py

```diff
@@ -46,3 +49,37 @@ def send_results(self, collected_results: list):
             logger.info('Scan: ' + raw_scan.tool_name + ' run at ' + scan_time)
             for issue in raw_scan.issues:
                 logger.info('Issue: ' + str(issue))
+
+
+
+
+    def load_files(self, protobuf, location):
+        """Given a protobuf object and a filesystem location, attempts to load all *.pb
+        files found in directories underneath the location into the protobuf object
+
+        :param protobuf: object expected to be found in the location
+        :param location: directory where protobuf objects are stored
+
+        :returns array of protobuf objects of the given type which were found at the location
+        :raise SyntaxError: If there are no .pb files found in location
+        """
+
+        logger.info('Searching for scan results')
+        collected_files = []
+
+        for filename in  Path(location).glob('**/*.pb'):
+            logger.info("Found file %s" % filename)
+            with open(filename, "rb") as f:
+                try:
+                    protobuf.ParseFromString(f.read())
+                    collected_files.append(copy.deepcopy(protobuf))
+
+                except DecodeError as e:
+                    logger.warning('Unable to parse file %s skipping because of: %s '%(filename,str(e)))
```

Comment:
Fix spacing

Reply:
This isn't fixed.

```python
logger.warning('Unable to parse file %s skipping because of: %s '%(filename,str(e)))
```

should be

```python
logger.warning('Unable to parse file %s skipping because of: %s ' % (filename,str(e)))
```

Reply:
resolved by following pylint's recommendation of passing variables as log arguments